### ISSUE TITLE
check_pdh and warning and critical params

### ISSUE THREAD

**Issue by brunomartins** [mickem/nscp#290]
I set up a check_pdh check that requires me to have a number with 12 digits as the threshold for the warning and critical parameters. From the documentation it appears that both the warning and critical parameters only accept either a float or an int.

Any chance support could be added for Long numbers :)

As an example, if I look at disk usage using Windows counters in bytes warning or critical parameters can only be set up to 2GB as that is the max for an int.

**Comment by mickem** [comment]
the "ints" should actually be unsigned long longs and the "floats" are doubles if you go by c++ types.
So I manage to get my filters to 9223372036854775807 (which is the largest value):

```
check_pdh "counter=\\Fysisk disk(0 C:)\\Disktid i procent" "warn=value > 9223372036854775807"
L        cli OK: OK: \Fysisk disk(0 C:)\Disktid i procent = 0
L        cli  Performance data: '\Fysisk disk(0 C:)\Disktid i procent_value'=0;9223372036854775808;0
```

But this was most likely fixed in 0.5.0

**Comment by mickem** [comment]
If you still have this issue in 0.5.1 please reopen this ticket.
